fhir-resource-tooling/src/main/java/com/ibm/cohort/tooling/fhir/MeasureImporter.java

```diff
@@ -273,6 +224,7 @@ public static void runWithArgs(String[] args, PrintStream out) throws Exception
 
 		if (arguments.isDisplayHelp) {
 			jc.usage();
+			resultCode = 1;
```

Comment:
Minor quesiton: Does a non-zero code make sense when a user provides the `-h` flag? Wouldn't displaying the help output be the correct behavior for the program?